### ISSUE TITLE
DDN FF63: Add initial support of scrollbar-width

### ISSUE THREAD

**Issue by estelle** [mdn/sprints#438]
https://bugzilla.mozilla.org/show_bug.cgi?id=1475033

**Comment by chrisdavidmills** [comment]
Please do; assigned to you.

**Comment by rachelandrew** [comment]
PR for the spec data: https://github.com/mdn/kumascript/pull/796

**Comment by rachelandrew** [comment]
PR for compat data: https://github.com/mdn/browser-compat-data/pull/2713

Once I have these bits in place I'll make the page.

**Comment by rachelandrew** [comment]
I'm blocked on this one as I don't have all the data in place, reviews of those PRs appreciated.

**Comment by chrisdavidmills** [comment]
All PRs now reviewed.

**Comment by chrisdavidmills** [comment]
@rachelandrew there is a production step needed to get all the new data appearing on MDN. This usually happens once a week, on a Thursday I think.

If you've submitted all the data, your PRs have been accepted, and the MDN pages/bugzilla bugs are updated, then we can call this DDN done.

**Comment by rachelandrew** [comment]
I need to update the bug and the release page - if everything is in progress then I'll do that. Just wanted to check I hadn't missed something.

**Comment by chrisdavidmills** [comment]
Nope; tis' all good.

**Comment by rachelandrew** [comment]
It appears to be currently behind a flag. Is it staying behind the flag for 63? The example on the page for example works in Nightly, but needs the `layout.css.scrollbar-width.enabled` flag.

I also note that the color properties from the same spec are enabled behind a flag too, so I wondered why we are adding this one and not those? See: https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Experimental_features#scrollbar-properties

**Comment by chrisdavidmills** [comment]
Nightly is on Fx64, so 63 will definitely be behind the flag.

We normally add features as we find bugs for them; I'm surprised we haven't found a bug for them as well. I've just done a search and can't seem to find anything. Would you be OK to add pages for those as well?

**Comment by rachelandrew** [comment]
absolutely, I also need to update the data to indicate it is behind a flag

**Comment by chrisdavidmills** [comment]
Perfect, thanks @rachelandrew !

**Comment by rachelandrew** [comment]
PR for compat data, adding flag info and adding the files for the color properties https://github.com/mdn/browser-compat-data/pull/2819

**Comment by rachelandrew** [comment]
Additional pages:

https://developer.mozilla.org/en-US/docs/Web/CSS/scrollbar-track-color
https://developer.mozilla.org/en-US/docs/Web/CSS/scrollbar-face-color

**Comment by rachelandrew** [comment]
I've updated the bug to let the engineer know these are done.
